pkg/virt-handler/isolation/detector.go

```diff
@@ -156,7 +156,7 @@ func (s *socketBasedIsolationDetector) AdjustResources(vm *v1.VirtualMachineInst
 // virt-launcher pod on the given VMI according to its spec.
 // Only VMI's with VFIO devices (e.g: SRIOV, GPU), SEV or RealTime workloads require QEMU process MEMLOCK adjustment.
```

Comment:
Do vDPA devices require it as well? I would update this comment with that information.

Reply:
Yes, vDPA devices also needs to adjust the memlock limit. Take a look at [this](https://patchew.org/Libvirt/20210323185052.961910-1-laine@redhat.com/).